data/quickstarts/openvino-inference-notebook.yaml

```diff
@@ -2,7 +2,7 @@ kind: ConsoleQuickStart
 metadata:
   name: openvino-inference-notebook
   annotations:
-    opendatahub.io/categories: 'AI/Machine learning,Getting started,Jupyter notebook,TensorFlow,Python,PyTorch'
+    opendatahub.io/categories: 'Getting started,Model optimization,Data preprocessing'
```

Comment:
I'd add "Notebook environemnts" category too, since openvino is used inside a Jupyter notebook

Reply:
Added, thanks Berto.